aboutPage.html

```diff
@@ -0,0 +1,140 @@
+        ></a>
+        <a id="infoTwiterLogoLink" href=""
+          ><i id="infoTwiterLogo" class="fab fa-twitter"></i
+        ></a>
+        <a href="infoLanguage">English</a>
+        <a href="infoMyPage">My Page</a>
+        <a href="infoLogOut">Logout</a>
+      </span>
+
+      <span id="desktopMenu">
+        <a href="index.html"
+          ><img id="navLogo" src="images/pageLogo.svg" alt="" class="logo"
+        /></a>
+        <a id="navAbout" href="aboutPage.html" target="_blank">About</a>
+        <a id="navProgram" href="index.html">Program</a>
+        <a id="navJoin" href="">Join</a>
+        <a id="navSponsor" href="">Sponsor</a>
+        <a id="navNews" href="">News</a>
+        <a id="navCCCamping" href="">CC Camping</a>
+      </span>
+    </nav>
+
+    <header id="header">
+      <h2 id="headerMainH2">"There is another way to learn"</h2>
+      <h1 id="headerH1">DIDACTIC ELECTRONICS LEARNING PROGRAM</h1>
+      <p id="aboutHeaderText1">
+        Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod
+        tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim
+        veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea
+        commodo consequat. Duis aute irure dolor in reprehenderit in voluptate
+        velit esse cillum dolore eu fugiat nulla pariatur. Excepteur sint
+        occaecat cupidatat non proident, sunt in culpa qui officia deserunt
+        mollit anim id est laborum.
+      </p>
+      <p id="headerText2">
+        Please contact us per email for any further questions about didactic
+        electronics
+        <br /><b> luispomare92@gmail.com</b>
+      </p>
+    </header>
+
+    <main id="aboutMain" class="flexColumn">
+      <h2 id="mainTittleOne" class="aboutMainTittles">
+        Didactic Electronics <br />
+        contribution
+      </h2>
+      <div id="mainUnderscoreOne" class="underscore"></div>
+      <p id="mainParagraphOne">
+        The Didactic electronics program has been created with the help of
+        thousands of students like you that are interested in this marvelous
+        world of knowledge.
+      </p>
+      <img src="images/pageLogo.svg" alt="" id="mainLogo" />
+
+      <hr />
+
+      <h2 id="mainTittleTwo">See success stories</h2>
+      <div id="mainUnderscoreTwo" class="underscore"></div>
+      <p id="mainParagraphTwo">
+        Take a look at testimonials from people who went through the program and
+        how it positively impacted their lives.
+      </p>
+      <div id="images" class="flexColumn">
+        <section id="projectOne" class="flexColumn"></section>
+        <section id="projectTwo" class="flexColumn">
+          <h2 id="projectTwoTittle">Procesing project</h2>
+          <p id="projectTwoParagraph">
+            A system that controls the functionality of a bottle-making machine.
+          </p>
+        </section>
+      </div>
+    </main>
+
+    <footer id="footer">
```

Comment:
- [ ] Great job working with the same background-color and style for the original design in your footer, let's make the footer much better by adding some `padding` and align-items in the center 

**Current mobile menu**           |  **Original design**
:-------------------------:|:-------------------------:
![](https://user-images.githubusercontent.com/80895497/153972970-1141ed2a-b00e-4be3-96d5-8b95b3b967d5.PNG)  |  ![](https://user-images.githubusercontent.com/80895497/153972962-77289bc0-2ed8-495c-922c-e567bc9f17f6.PNG)

Reply:
👍🏽 Good now.